src/index_find_regions.js

```diff
@@ -17,7 +17,8 @@ exports.handler = (event, context, callback) => {
     }
     event.awsevents.Credentials = credentials;
     credentials = new Buffer(JSON.stringify(credentials)).toString('base64');
-    event.cloudtrail.headers.Credentials = credentials;
```

Comment:
Why are we removing the credentials

Reply:
Now We don't require these credentials here as they were there to create credentials for each region.

Reply:
I added this line in the "index_health_stack.js"

Reply:
That would be needed for config rules